internal/pkg/cli/env_list_test.go

```diff
@@ -23,10 +23,34 @@ func TestEnvList_Execute(t *testing.T) {
 	defer ctrl.Finish()
 
 	testCases := map[string]struct {
-		listOpts    ListEnvOpts
-		mocking     func()
-		expectedErr error
+		listOpts        ListEnvOpts
+		mocking         func()
+		expectedErr     error
+		expectedContent string
 	}{
+		"with json envs": {
+			listOpts: ListEnvOpts{
+				ShouldOutputJSON: true,
+				manager:          mockEnvStore,
+				projectGetter:    mockProjectStore,
+				GlobalOpts: &GlobalOpts{
+					projectName: "coolproject",
+				},
+			},
+			mocking: func() {
+				mockProjectStore.EXPECT().
+					GetProject(gomock.Eq("coolproject")).
+					Return(&archer.Project{}, nil)
+				mockEnvStore.
+					EXPECT().
+					ListEnvironments(gomock.Eq("coolproject")).
+					Return([]*archer.Environment{
+						{Name: "test"},
+						{Name: "test2"},
+					}, nil)
+			},
+			expectedContent: `{"environments":[{"project":"","name":"test","region":"","accountID":"","prod":false,"registryURL":"","managerRoleARN":""},{"project":"","name":"test2","region":"","accountID":"","prod":false,"registryURL":"","managerRoleARN":""}]}` + "\n",
```

Comment:
should the expected project name be empty?

Reply:
Yeah, because the dummy response from the mock only has a name for the environment and not the project :P